### PR TITLE
ldap: improve documentation

### PR DESCRIPTION
**Pull request by Habbie**
- link to relevant PR
- add secondary schema
- write brief installation instructions

NOTE: I did not test the installation instructions end to end. Volunteers welcome.

### CODE REVIEW

File: docs/markdown/authoritative/backend-ldap.md [hunk 1]
```diff
@@ -46,9 +46,78 @@ which are required by the LDAP server to check the validity of entries
...
+index   associatedDomain  eq
+```
+
+You should now be able to import zones from a given `named.conf` with a line such as:
+
+```
+zone2ldap --dnsttl=yes --basedn=ou=dns,o=power --layout=$layout --named-conf=named.conf | ldapmodify -D uid=testuser,o=power -w secret -c
+```
+
+Make sure to replace $layout with your choice of database layout, and mirror this in the `ldap-method` setting in your `pdns.conf`.
```
Comment (jpmens) [inline docs/markdown/authoritative/backend-ldap.md]: Maybe link to documentation for "database layout"?

Comment (Habbie) [inline docs/markdown/authoritative/backend-ldap.md]: done in extra commit, thanks

**Comment by pieterlexis** [comment]
is this still relevant with #4477 on the table?

**Comment by goestreicher** [comment]
I have another PR lined up for the LDAP backend, not open yet as it depends on #4477, that will address this issue. You can check the doc that will be part of this PR here: https://github.com/KamagoNet/pdns/blob/ldap-dnssec/docs/markdown/authoritative/backend-ldap.md

**Pull request closed without merging**